rdr_service/api/genomic_cloud_tasks_api.py

```diff
@@ -101,13 +138,7 @@ def post(self):
 
             logging.info(f'{message} task data: {task_data}')
 
-            # Call pipeline function
-            genomic_pipeline.execute_genomic_manifest_file_pipeline(task_data)
-
-            self.create_cloud_record()
-
-        logging.info('Complete.')
-        return {"success": True}
```

Comment:
For Google to show that it was successful, would this still need to return the success response? Should this return the result of the `execute_manifest_ingestion` call?